airbyte-integrations/connectors/destination-s3-data-lake/metadata.yaml

```
@@ -26,7 +26,7 @@ data:
             alias: airbyte-connector-testing-secret-store
   connectorType: destination
   definitionId: 716ca874-520b-4902-9f80-9fad66754b89
-  dockerImageTag: 0.2.23
+  dockerImageTag: 0.2.24
```

I forgot about this with the json stringify stuff, but let's do a bigger version bump for breaking change :P
```suggestion
  dockerImageTag: 0.3.0
```

---

[243ec34](https://github.com/airbytehq/airbyte/pull/52639/commits/243ec34f4bef5e2cfff102a5b404bf8124eaea23)